### DISCUSSION TITLE
Is there a way to organize partials into folders?

### DISCUSSION THREAD

**Discussion by joan-medina-roca** [GetPublii/Publii#1785] [Help] [answered]
Hello fellow Publii enthusiasts,

I have been working on a website for the last few months, and the result is I have a folder full of partials...

Right now, I have about 50 and there's a lot more to come, since partials are super handy.

However, partials are becoming a lot harder to navigate since there are so many of them. **I wonder if there is a way to organize partials (except for changing the name, of course), into subfolders**.

Or... any other idea you might have that would mitigate this problem would be very welcome.

Thanks everyone!

**Comment by bobmitro** [comment]
There is no possible right now. To stay organized, use clear naming conventions for your partials, such as `header-post.hbs` or `header-main.hbs`, to indicate where each partial is used.

**Comment by candidexmedia** [comment]
That would be handy! It would be great if Publii was able to find them regardless of the sub-folder they're in, so long as they're in the `partials` folder

**Comment by dziudek** [comment]
@joan-medina-roca @candidexmedia - I will try to add this feature in v.0.46.3, because in fact for more complicated themes it can be useful for code organization :)

**Reply by candidexmedia** [reply]
Awesome!!

**Reply by joan-medina-roca** [reply]
wow wow wow!!!

**Comment by joan-medina-roca** [comment, marked as answer]
Oh guys! I see you implemented this in the newest update. Genius! Thank you so much :D

**Reply by dziudek** [reply]
@joan-medina-roca - yes, sorry I forgot to inform you in this thread :)
